main.go

```diff
@@ -61,7 +61,8 @@ func run(ctx context.Context, wg *sync.WaitGroup, globalCollectionOpts state.Col
 		prefixedLogger := logger.WithPrefix(server.SectionName)
 		prefixedLogger.PrintVerbose("Identified as api_system_type: %s, api_system_scope: %s, api_system_id: %s", server.SystemType, server.SystemScope, server.SystemID)
 
-		conf.Servers[idx].HTTPClient = config.CreateHTTPClient(server)
+		conf.Servers[idx].HTTPClient = config.CreateHTTPClient(server, false)
+		conf.Servers[idx].HTTPClientWithRetry = config.CreateHTTPClient(server, true)
```

Comment:
We could pass the `prefixedLogger` here so HTTP error logs have the right prefix (I suspect we may also need that for the existing HTTP client?)